extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -557,7 +557,7 @@ export class ApplicationInsights extends BaseTelemetryPlugin implements IAppInsi
                 measurements
             };
 
-            this.sendPageViewInternal(pageViewItem);
+            this.sendPageViewInternal(pageViewItem, properties);
```

Comment:
I think we need a test for this. Wouldn't we also want to send measurements as well?

Reply:
Good point, added some test, and double checked code. The reason `properties.refUri` was overwritten is because [this line](https://github.com/microsoft/ApplicationInsights-JS/blob/ebb1b6adc1b56fbcb4f086ef2baacad5cab1d7af/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts#L263). `measurements` filed is passed through `pageView` param then in saved in `baseData` [here](https://github.com/microsoft/ApplicationInsights-JS/blob/ebb1b6adc1b56fbcb4f086ef2baacad5cab1d7af/shared/AppInsightsCommon/src/TelemetryItemCreator.ts#L44).